AUTHORS_RESUBMITTING_UNDER_LGPL_LICENSE.md

```diff
@@ -0,0 +1,43 @@
+# List of authors resubmitting their contributions under LGPL-3.0-or-later
+
+VOLK is going to migrating from GNU General Public License v3.0 or later (GPL-3.0-or-later)
```

Comment:
"is going to migrating" -> "is migrating" ... or something along those lines

Reply:
```suggestion
VOLK is migrating from GNU General Public License v3.0 or later (GPL-3.0-or-later)
```